app/org/sagebionetworks/bridge/models/surveys/SurveyRule.java

```diff
@@ -133,12 +173,29 @@ public Builder withEndSurvey(Boolean endSurvey) {
             }
             return this;
         }
+        public Builder withDisplayIf(Boolean displayIf) {
+            if (Boolean.TRUE.equals(displayIf)) {
+                this.displayIf = displayIf;
+            }
```

Comment:
I haven't really ever called this method more than once because it's in the builder... but I can add that if you think it's a problem down the road.